seqr/views/apis/variant_search_api_tests.py

```diff
@@ -516,7 +515,7 @@ class AnvilVariantSearchAPITest(AnvilAuthenticationTestCase, VariantSearchAPITes
 
     def test_query_variants(self, *args):
         super(AnvilVariantSearchAPITest, self).test_query_variants(*args)
-        assert_no_list_ws_has_al(self, 9)
+        assert_no_list_ws_has_al(self, 10)
```

Comment:
is this a merge issue/ because you are behind dev, or is this reverting something back that we don't want?

Reply:
It is because `dev` was not updated when the changes were committed. It will disappear after I refresh the base branch.